java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -478,7 +478,7 @@ public ColumnVector makeListFromOffsets(long rows, ColumnView offsets) {
     return new ColumnVector(makeListFromOffsets(getNativeView(), offsets.getNativeView(), rows));
   }
 
-  /**
+   /**
```

Comment:
Indentation looks off here, this line should not have changed.

Reply:
Just fixed it. Thanks.

Reply:
It's not fixed, still showing up as a diff.

Reply:
Hah, it was off by 2 spaces. I fixed by deleting 1 space :smiley: